styles.css

```diff
@@ -164,6 +164,48 @@ h3 {
   margin: 0 0 5px 0;
 }
 
+.avatar {
+  background-color: none;
+  border-radius: 50%;
+  border: 5px double white;
+  margin-left: -122px;
+  width: 9%;
+  z-index: 1;
+}
+
+.avatar1 {
+  background-image: url('assets/alien1.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
```

Comment:
Suggest adding lines 178-180 to .avatar class, these are used in all three avatars.